core/shared/src/main/scala/zio/BootstrapRuntime.scala

```diff
@@ -19,7 +19,7 @@ package zio
 import zio.internal.Platform
 
 trait BootstrapRuntime extends Runtime[ZEnv] {
-  def environment: ZEnv = ZEnv.Services.live
+  val environment: ZEnv = ZEnv.Services.live
```

Comment:
```suggestion
  def environment: ZEnv = ZEnv.Services.live
```
Need to be careful changing `def` to `val` in traits that can be extended because it makes it no longer possible to implement as a `def` or `lazy val` in subtypes and can potentially cause early initialization issues.